crates/typst/src/layout/spacing.rs

```diff
@@ -18,7 +18,8 @@ use crate::utils::Numeric;
 ///
 /// # Mathematical Spacing { #math-spacing }
 /// In [mathematical formulas]($category/math), you can additionally use these
-/// constants to add spacing between elements: `thin`, `med`, `thick`, `quad`, `wide`.
+/// constants to add spacing between elements: `thin` (1/6em), `med`(2/9em),
```

Comment:
```suggestion
/// constants to add spacing between elements: `thin` (1/6 em), `med`(2/9 em),
```
Without a space, this can be interpreted as Typst code `1 / 6em`, which is invalid. It may be clearer with a space. Or maybe U+2159 ⅙ could be used, but I don't think there are equivalent characters for 2/9 and 5/18.

Reply:
I originally wanted to do that (via fraction slash and narrow no-break space) but decided against it. There’s no guarantee the fractions display correctly on all devices, and the devs don’t seem to like having a space between number and unit.

Reply:
[Resolved via Discord.](https://discord.com/channels/1054443721975922748/1088371919725793360/1272514010985795648)